ForgeUnity/Assets/BeardedManStudios/Scripts/Networking/Forge/Networking/Objects/NetworkObject.cs

```diff
@@ -208,6 +208,17 @@ public abstract class NetworkObject
 		/// </summary>
 		public NetworkingPlayer Owner { get; private set; }
 
+		/// <summary>
+		/// A reference to the player who can send RPCs this network object
+		/// </summary>		
+		public NetworkingPlayer Master { get; private set; }
+
+		/// <summary>
+		/// If set to true on the server, anyone can send RPCs to this network object,
+		/// otherwise only the Owner and the Master can
+		/// </summary>
+		public bool AllowPublicRpcs { get; set; }
```

Comment:
Please set this to true to keep current functionality.

Reply:
The main idea was to "force" people to go for an authoritative design as it's extremely easy for people to mistake the authority of RPCs.

> Often you will hear the term authoritative server and in most cases, it is used to mean authoritative game logic. So, we would like to first go over the difference between the two as we often state that Forge Networking (including Remastered) **is authoritative server by design**.

Reply:
Please don't introduce BC breaks. Over time this can be changed to be false by default, but initially it should keep current functionality.